src/store/index.js

```diff
@@ -70,6 +87,17 @@ const selectors = {
 	isInitialStateLoaded: ( state ) => {
 		return state.isInitialStateLoaded;
 	},
+	getPluginsArray: (state) => {
```

Comment:
now that we have determined names for each of the IDE areas, possibly this could be renamed to something like `getActivityBarPlugins` instead of just `plugins` 🤔

Reply:
I agree. Feel free to make that change here if you feel inclined to, otherwise can make a separate issue to scrub the whole codebase for any other possible renames before moving on.

Reply:
@josephfusco let's follow-up on this. 

I think we'll see some other patterns emerge re: naming things, file organization, etc as we start implementing the other "pluggable" areas